src/components/dialog/dialog.atoms.tsx

```diff
@@ -0,0 +1,36 @@
+import { forwardRef } from 'react'
+
+import { DialogBodyProps, DialogFooterProps, DialogContainerProps, DialogTitleProps } from './types'
+import { ElDialogBody, ElDialogContainer, ElDialogFooter, ElDialogTitle } from './style'
+
+const Container = forwardRef<HTMLDialogElement, DialogContainerProps>(({ children, ...rest }, ref) => {
+  return (
+    <ElDialogContainer {...rest} ref={ref}>
+      {children}
+    </ElDialogContainer>
+  )
+})
```

Comment:
> Is this a requirement from the Figma spec? If not, I don't think we should do it.

No they're not, alright I'll keep them in my mind

> What's hacky with the fade out transition? I don't see any further context about this here or in styles.ts where the styled dialog element is defined 🤔

By looking the Dialog component in the React UI Core, they have the fadeIn and fadeOut transition after some MouseEvent event occurs; The outlined hacky part in here is related with the hand of JavaScript, react user would really happy for this since we could give the bundled package, but for non react user might need to extend their responsibility to add the `fadeOut` transition in their dialog version.

**thought:** I'm thinking that we could use [Animation](https://developer.mozilla.org/en-US/docs/Web/API/Animation) and [KeyframeEffect](https://developer.mozilla.org/en-US/docs/Web/API/KeyframeEffect) web APIs to achieve this